front/config.form.php

```diff
@@ -30,18 +30,18 @@
  *  --------------------------------------------------------------------------
  */
 
-include ("../../../inc/includes.php");
```

Comment:
I assume the modifications to the PHP files come from the php-cs-fixer command. It’s not related to the subject of this PR, so it should be done in a separate dedicated PR.